### PR TITLE
Suru++ Asprómauros plugin (#232)

### PR DESCRIPTION
**Pull request by gusbemacbe**
Hello @actionless!

As you suggested me one of two options in the #232, I opted for the 2nd option.

### CODE REVIEW

**Comment by actionless** [comment]
oh man, you've got few minutes late after 1.12 release :D

**Comment by gusbemacbe** [comment]
Too bad that I was too late.

File: .gitmodules [hunk 1]
```diff
@@ -32,3 +32,8 @@
 [submodule "suru-plus"]
 	path = plugins/icons_suruplus/suru-plus
 	url = https://github.com/gusbemacbe/suru-plus.git
+[submodule "suru-plus-aspromauros"]
+	path = plugins/icons_suruplus_aspromauros/suru-plus-aspromauros
+	url = https://github.com/gusbemacbe/suru-plus-aspromauros.git
+
+
```
Comment (actionless) [inline .gitmodules]: please remove empty space

**Comment by actionless** [comment]
you couldn't know, but we can push 1.12.1 after testing this

File: .gitmodules (hunk 1 above)
Comment (gusbemacbe) [reply .gitmodules]: Ready! If it goes something, I will rebase and then git pull upstream.

I understood how the rebase works. Rebase is like downgrade and is like as if the starship USS Enterprise travelled through the alternate timeline.

**Comment by actionless** [comment]
i've just tried -- on preview in the app icons are looking the same as in normal Suru++

could it be what you forgot to update the preview icons?

**Comment by gusbemacbe** [comment]
> could it be what you forgot to update the preview icons?

Are you referring to the SVGA files templates?

**Comment by actionless** [comment]
./plugins/icons_suruplus_aspromauros/icon_previews/user-home.svg.template

**Comment by actionless** [comment]
thanks a lot!

**Comment by actionless** [comment]
i've updated `oomox-git` AUR package

**Comment by gusbemacbe** [comment]
I will test. But I installed manually instead of installing via AUR. Would you like I test via AUR?

**Comment by gusbemacbe** [comment]
I have just tested, it worked everywhere.

**Comment by actionless** [comment]
i just found a problem with it, it have unused options in the plugin:

2019-03-10--1552204770_1424x529_scrot

**Comment by gusbemacbe** [comment]
Do you mean "Light Base (Folders)", "Medium" and "Dark"? I forgot of it. Sorry!

**Comment by gusbemacbe** [comment]
I am going to fix it.

**Pull request merged**